templates/user/openid_manage.mako

```diff
@@ -1,6 +1,29 @@
 <%inherit file="/base.mako"/>
-<%namespace file="login.mako" import="render_openid_form" />
 <%def name="body()">
     <h2>Associate more OpenIDs</h2>
     ${render_openid_form( redirect, True, openid_providers )}
-</%def>
\ No newline at end of file
+</%def>
+
+<%def name="render_openid_form( redirect, auto_associate, openid_providers )">
+    <div class="toolForm">
+        <div class="toolFormTitle">OpenID Login</div>
```

Comment:
Do you think putting a text such as "this feature will be deprecated in 19.05 release" would be useful?